<launch>
	
  <arg name="odom_topic" default="/ekf/ekf_odom"/>
				
  <node pkg="px4ctrl" type="px4ctrl_node" name="px4ctrl" output="screen">
			<remap from="~odom" to="$(arg odom_topic)" />
			<remap from="~cmd" to="/position_cmd" />

        <rosparam command="load" file="$(find px4ctrl)/config/ctrl_param_traj8.yaml" />
	</node>
 
</launch>

